README.md

```diff
@@ -124,8 +125,18 @@ In order to facilitate direct comparisons between results published in the liter
 
 ```python
 datamodule = InriaAerialImageLabelingDataModule(root="...", batch_size=64, num_workers=6)
-task = SemanticSegmentationTask(segmentation_model="unet", encoder_weights="imagenet", learning_rate=0.1)
-trainer = Trainer(gpus=1, default_root_dir="...")
```

Comment:
`gpus=1` is replaced by `accelerator="gpu", devices=1`, but the correct device is now chosen automatically by default.